kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -188,6 +191,10 @@ private MultipartUpload newMultipartUpload() throws IOException {
         newObjectMetadata()
     ).withCannedACL(cannedAcl);
 
+    if (ssea.equals("aws:kms") && StringUtils.isNotBlank(awsKmsKeyId)) {
```

Comment:
This should use `SSEAlgorithm.KMS.getAlgorithm()` rather than the literal string "aws:kms"